plugins/community/timostamm-protobuf-ts/v2.8.2/buf.plugin.yaml

```diff
@@ -0,0 +1,17 @@
+version: v1
+name: buf.build/community/timostamm-protobuf-ts
+plugin_version: v2.8.2
+source_url: https://github.com/timostamm/protobuf-ts
+description: Generates Protobuf and RPC for TypeScript.
+output_languages:
+  - typescript
+registry:
```

Comment:
Did you test remote packages support?

Reply:
For best remote packages support, the option `output_javascript` should be set. This will output JavaScript files along with TypeScript declaration files. This is very heavy on resources though.